CONTRIBUTING.md

```diff
@@ -43,8 +43,19 @@ Once you have built your template file or files, the commit message should have
 
 * for template modifications without a version change, use ```<pkgname>: <reason>``` ([example](https://github.com/void-linux/void-packages/commit/8b68d6bf1eb997cd5e7c095acd040e2c5379c91d)).
 
+* for package removals, use ```<pkgname>: remove package``` ([example](https://github.com/void-linux/void-packages/commit/83784632d94deee5d038c8e1c4c1dffa922fca21)).
+
+* for `common/shlibs` modifications, use `common/shlibs: <pkgname>` ([example](https://github.com/void-linux/void-packages/commit/613651c91811cb4fd2e1a6be701c87072d759a9f)).
+
 If you want to describe your changes in more detail, add an empty line followed by those details ([example](https://github.com/void-linux/void-packages/commit/f1c45a502086ba1952f23ace9084a870ce437bc6)).
 
+Pull requests are automatically submitted for Continuous Integration (CI) testing to ensure packages build on various combinations of libc and architectures.
+Packages that take longer than 50 minutes to build (for example, Firefox or the Linux kernel) will fail CI and should include `[ci skip]` in the last line of the last commit message in the PR to avoid wasting CI builder time.
+Use your best judgment on build times based on your local building experience. If you skip CI when submitting a PR, please build and cross-build for a variety of architectures locally, with both glibc and musl, and note your local results in PR comments.
+Make sure to cover 64-bit and 32-bit architectures
+
+It should be noted that packages should also be runtime tested. Building them with the `-Q` flag or with the `XBPS_CHECK_PKGS=yes` variable in `etc/conf` is encouraged.
```

Comment:
Quick manual test of built software are encouraged too. For new packages it is requirement.

Reply:
```
It should be noted that runtime testing packages, as well as building them with the `-Q` flag or with the `XBPS_CHECK_PKGS=yes` variable in `etc/conf` is encouraged.
New packages are required to have been runtime tested.
```

What do you think?

Reply:
Runtime testing of packages and building with the `-Q` flag or with `XBPS_CHECK_PKGS=yes` set in the environment or `etc/conf` are strongly encouraged. New packages will not be accepted unless they have been runtime tested.